code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -244,6 +244,12 @@
 		"<span class='userdanger'>You feel a powerful shock coursing through your body!</span>", \
 		"<span class='italics'>You hear a heavy electrical crack.</span>" \
 		)
+	if(iscarbon(pulling) && !illusion && source != pulling)
+		var/mob/living/carbon/human/H = pulling

```

Comment:
If you are checking for carbon then you type it as carbon

Reply:
Humans will still use their own electrocute_act so don't do anything special for them unless you have to call a var/proc specific to humans.

Reply:
You.. are right, they do. something something wrapping head around type casting grumble grumble.